actuators/Magnetorquer.cpp

```diff
@@ -50,16 +32,16 @@ Vector3 Magnetorquer::getTorque(VectorXd voltages, Quaternion q, Vector3 magneti
     Eigen::VectorXd max_applied_power = power.cwiseAbs();
 
     for (int i=0; i<num_MTBs; i++) {
-        assert(max_current(i) <= max_current_rating(i));
-        assert(max_applied_voltage(i) <= max_voltage(i));
-        assert(max_applied_power(i) <= max_power(i));
+        assert(max_current(i) <= max_current_rating);
+        assert(max_applied_voltage(i) <= max_voltage);
+        assert(max_applied_power(i) <= max_power);
     }
 
     //auto dipole_moments = N_per_face*A_cross*
     Vector3 dipole_moment;
     MatrixXd torque = MatrixXd::Zero(3, num_MTBs);
     for (int i=0; i<num_MTBs; i++) {
-        dipole_moment = N_per_face(i)*currents(i)*A_cross(i)*G_mtb_b.col(i);
```

Comment:
originally I had made these separate so we could disperse these ratios differently for each magnetorquer.